<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWarehouseComponentDialog</class>
 <widget class="QDialog" name="AddWarehouseComponentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>ИД комплектующего</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputComponentId"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>ИД склада</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputWarehouseId"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Количество комплектующих</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputAmount"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>200</number>
       </property>
       <item>
        <widget class="QPushButton" name="buttonAddWarehouseComponent">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
background-color: rgb(250, 250, 250);
border: 1px solid black;
padding: 3px;
}

QPushButton:pressed
{
background-color: rgb(240, 240, 240);
}

QPushButton:hover:!pressed
{
background-color: rgb(240, 240, 240);
}</string>
         </property>
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonCloseDialog">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
background-color: rgb(250, 250, 250);
border: 1px solid black;
padding: 3px;
}

QPushButton:pressed
{
background-color: rgb(240, 240, 240);
}

QPushButton:hover:!pressed
{
background-color: rgb(240, 240, 240);
}</string>
         </property>
         <property name="text">
          <string>Закрыть</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>AddWarehouseComponentDialog</sender>
   <signal>finished(int)</signal>
   <receiver>AddWarehouseComponentDialog</receiver>
   <slot>deleteLater()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCloseDialog</sender>
   <signal>clicked()</signal>
   <receiver>AddWarehouseComponentDialog</receiver>
   <slot>deleteLater()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
